src/main/kotlin/racingcar/view/PromptServiceImpl.kt

```diff
@@ -0,0 +1,27 @@
+package racingcar.view
+
+import racingcar.view.Message.RACING_CANDIDATE_NAME_INPUT
+import racingcar.view.Message.RACING_MOVE_INPUT
+
+class PromptServiceImpl(
+    private val inputView: InputView,
+    private val resultView: ResultView
+) : PromptService {
+
+    override fun getNameOfCars(assignedNames: String?): List<String> {
+        println(RACING_CANDIDATE_NAME_INPUT)
+
+        val names = assignedNames ?: inputView.readInput()
+        return names.split(",").map { it.trim() }
+    }
+
+    override fun getNumberOfMoves(assignedNumber: Int?): Int {
+        println(RACING_MOVE_INPUT)
+
+        return assignedNumber ?: inputView.readInput().toInt()
+    }
+
+    override fun showResult(result: List<Map<String, Int>>) {
```

Comment:
```suggestion
    override fun showResult(result: List<CarRacingResult>) {
```
만들어두신 typealias를 활용해보시면 어떨까요?